lib/stat_tracker.rb

```diff
@@ -4,11 +4,6 @@ class StatTracker
   attr_reader :locations, :data
 
   def initialize(game_path, team_path, game_teams_path)
```

Comment:
this might be for later but we can possibly make a parent class for the stat_tracker and pass in these arguments